src/simulation.hpp

```diff
@@ -901,14 +900,18 @@ template <typename problem_t> void AMRSimulation<problem_t>::ellipticSolveAllLev
 
 		// solve Poisson equation with open b.c. using the method of James (1977)
 		amrex::Vector<amrex::MultiFab> phi(finest_level + 1);
+		// define another fab to store phi
+		amrex::Vector<amrex::MultiFab> phi_data(finest_level + 1);
```

Comment:
If you are defining `phi_data` as part of the AMRSimulation class, you don't want to re-define it here as a local variable, because then the local variable will take precedence over the class variable, and the class variable won't be updated (or even initialized!).

Reply:
I think this will fix the segfault.